.eleventy.js

```diff
@@ -670,25 +670,30 @@ module.exports = function(eleventyConfig) {
     }
 
     markdownLib.renderer.rules.link_open = (tokens, idx, options, env, self) => {
-    const hrefIndex = tokens[idx].attrIndex('href');
-    if (hrefIndex >= 0) {
-        let href = tokens[idx].attrs[hrefIndex][1];
-        let classIndex = tokens[idx].attrIndex('class');
-        
-        // Exclude the link if it has the class 'header-anchor'
-        if (classIndex >= 0 && tokens[idx].attrs[classIndex][1] === 'header-anchor') {
-            return self.renderToken(tokens, idx, options);
-        }
-
-        // Ensure the URL has a trailing slash, but do not update if it contains a '#' or ends with '.md'
-        if (!href.endsWith('/') && !href.includes('#') && !href.endsWith('.md')) {
-            href += '/';
+        const hrefIndex = tokens[idx].attrIndex('href');
+        if (hrefIndex >= 0) {
+            let href = tokens[idx].attrs[hrefIndex][1];
+            const classIndex = tokens[idx].attrIndex('class');
+    
+            // Exclude the link if it has the class 'header-anchor'
+            if (classIndex >= 0 && tokens[idx].attrs[classIndex][1] === 'header-anchor') {
+                return self.renderToken(tokens, idx, options);
+            }
+    
+            // Ensure the URL has a trailing slash, but do not update if it contains a '#' or ends with '.md' or https
+            if (!href.endsWith('/') && !href.includes('#') && !href.endsWith('.md') && !href.includes('https')) {
+                href += '/';
+            }
+    
+            // make sure to update the flowfuse signup URL
+            if (href.includes('https://app.flowfuse.com/')) {
+                href += '/';
```

Comment:
@sumitshinde-84 isn't this going to be adding an extra `/` regardless of whether it already ends with `/`? You've added this outside of any check to guard against that.

Reply:
good point - I miss that

Reply:
Yes, I should first check if it already has the trailing slash or not, but I also want to make sure that the host, like app.flowfuse.com, will not have the slash added at the end.

Reply:
i will fix it now !